client/webserver/site/src/js/wallets.ts

```diff
@@ -232,6 +233,10 @@ export default class WalletsPage extends BasePage {
     Doc.hide(this.page.forms)
   }
 
+  copyAddress () {
+    navigator.clipboard.writeText(this.page.depositAddress.textContent || '')
```

Comment:
Looks good.  https://caniuse.com/mdn-api_clipboard_writetext
I'm assuming `'click'` counts as a "user gesture event handlers such as pointerdown or pointerup." so it will work on Safari.  Can anyone test on Safari?

Reply:
It works on Safari. I put the icon inside the box.. I think it looks better.